packages/edit-navigation/src/components/editor/style.scss

```diff
@@ -103,7 +103,6 @@
 
 	// Add Buttons
 	.block-editor-button-block-appender.block-list-appender__toggle {
-		margin: 0 0 0 $grid-unit-20;
```

Comment:
Was this change intentional?

Reply:
I thought it might be related to the test failure, but nope. I've dropped the commit from this branch.